### ISSUE TITLE
Evaluation

### ISSUE THREAD

**Issue by tdt98** [lgw863/ACSA-generation#5]
Thank you for sharing your great work, I would like to ask why dont you shift tokens to the right for decoder_input_ids  but you use the orgiginal labels ids in test_MAMS.py?
` output_ids = tokenizer(target_list, return_tensors='pt', padding=True, truncation=True)['input_ids']
        with torch.no_grad():
            output = model(input_ids=input_ids.to(device), decoder_input_ids=output_ids.to(device))[0]`
